lib/sync_checker/formats/edition_base.rb

```diff
@@ -66,7 +66,8 @@ def checks_for_draft(locale)
               expected_details_hash(edition_expected_in_draft)
             end
           ),
-          Checks::TranslationsCheck.new(edition_expected_in_draft.available_locales)
+          Checks::TranslationsCheck.new(edition_expected_in_draft.available_locales),
+          Checks::HttpStatusCheck.new(edition_expected_in_draft)
```

Comment:
It's not clear why we'd initialise this with the edition as we don't need to know anything about the edition, we just need a response object that is provided when the check is called.